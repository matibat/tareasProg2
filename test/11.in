son_iguales (1,a) (1,a)
son_iguales (1,a b) (2,a b)
son_iguales (1,a) (1, b)
son_iguales (2,a) (2, a)
combinar_info (10,abc) (20,cd)
combinar_info (50,abc -) (-30, - cd)
Fin
