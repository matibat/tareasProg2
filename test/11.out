1>Son iguales.
2>No son iguales.
3>No son iguales.
4>No son iguales.
5>El info combinado es (30,abccd).
6>El info combinado es (20,abc - - cd).
7>Fin.
